README.md

```diff
@@ -53,6 +53,13 @@ how to use AWS. The AWS Construct Library aims to reduce the complexity and
 glue-logic required when integrating various AWS services to achieve your goals
 on AWS.
 
+Modules in the AWS Construct Library are designated Experimental while we build
```

Comment:
Do we have a link to a resource which outlines which modules are experimental and which are stable?

Reply:
I don't know of such a page. I think one could be generated in the API Reference (just a small matter of programming).

Reply:
I'll at least mention that the Overview page of each module in the API Reference has this info.